decidim-verifications/lib/decidim/verifications/default_action_authorizer.rb

```diff
@@ -9,7 +9,7 @@ class DefaultActionAuthorizer
       # authorization - The existing authorization record to be evaluated. Can be nil.
       # options       - A hash with options related only to the current authorization process.
       # component     - The component where the authorization is taking place.
-      # resouce       - The resource where the authorization is taking place. Can be nil.
+      # resource      - The resource where the authorization is taking place. Can be nil.
```

Comment:
My scripts don't automatically fix up whitespace. I try to catch these things before I make PRs, but if I miss any, I'm sorry, just let me know, and I'll fix them up as I did here (just caught it recently even though it was composed as part of the original very-long series).

Reply:
We really appreciate the effort. The spell check it was long overdue.